### ISSUE TITLE
[UI] Basic Next.js scaffold

### ISSUE THREAD

**Issue by davidlhw** [is442oop/portfolio-analyzer-web-ui#1]
*TL;DR*: Build a basic Next.js app, following the [Next.js quickstart guide](https://nextjs.org/docs/getting-started/installation)

## Context

This project will use Next.js (serverless) as the web frontend.

## Features

In this iteration, we will be focusing on shipping a baseline product that we can build on top of in later iterations.
### Core

- A reachable endpoint at `http://localhost:3000/` that redirects to the `/overview` route, and shows the following screen:

![image](https://github.com/is442oop/portfolio-analyzer-web-ui/assets/13061926/382df596-4d7b-479b-aae8-e0df4b1306ec)

## Implementation

- [Next.js quickstart guide](https://nextjs.org/docs/getting-started/installation)
